plugin/src/main/java/com/denizenscript/denizen/npc/traits/SleepingTrait.java

```diff
@@ -58,12 +57,7 @@ else if (npc.getEntity() instanceof Player) {
                 ((Player) npc.getEntity()).sleep(bedLocation.clone(), true);
             }
             else {
-                if (NMSHandler.getVersion().isAtLeast(NMSVersion.v1_17)) {

```

Comment:
As noted on discord, given that the only special case here is a very old version, better to not touch the 1.16 special case unless you're 100% confident this change is entirely correct.

Reply:
Reverted